### ISSUE TITLE
Seeing more instances of 'we have no imagery' on Gallery

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#2995]
##### Brief description of problem/feature
Lately I've been seeing more instances of the "we have no imagery here" message in Gallery. And this is happening when our current method of checking if imagery exists says that there _is_ imagery. So maybe this method no longer works as well as it used to?

![Screenshot from 2022-08-18 12-04-49](https://user-images.githubusercontent.com/6518824/185474728-eec395bc-89d8-44dd-bd6f-1c77efcaaaab.png)

I think that this is happening more in Gallery where we are getting static images, but I haven't tested enough yet to know if it's happening as frequently on the validation page. I've also been seeing it a lot more locally than on production...

**Comment by misaugstad** [comment]
Yep, it looks like we are getting a valid response code 200 no matter what, and are just getting a blank image now instead of any sort of error shown in the response code. We've had methods we've used to deal with this in the pano-tools repo, so I'll look into that as well as other ways to check for imagery.

**Comment by jonfroehlich** [comment]
Interesting. I just saw this too when I tried to test out your new "mistakes" UI on the dashboard. Here's the Seattle test server:
![image](https://user-images.githubusercontent.com/1621749/185501176-21d52602-9639-4db7-97c9-4d633ec4aa2b.png)

But Oradell test seems fine. Note that **two** of the three labels are actually correct (if not all 3!). Argh!
![image](https://user-images.githubusercontent.com/1621749/185501295-2fb1946a-f535-429e-89d5-5c8e04965274.png)
